resources/site-packages/quasar/navigation.py

```diff
@@ -91,7 +91,7 @@ def run(url_suffix=""):
 
     try:
         data = _json(url)
-    except:
+    except urllib2.HTTPError, e:
```

Comment:
No need to check for specific errors since we're just returning. Just catch 'em all. Assigning `e` also causes a linting error:
```
navigation.py:94:31: F841 local variable 'e' is assigned to but never used
```